src/main/java/hudson/plugins/ansicolor/ColorConsoleAnnotator.java

```diff
@@ -102,20 +128,29 @@ public void emitHtml(String html) {
                     }
                     incoming.write(c);
                 }
+                nextOpenTags = ansiOs.getOpenTags();
+                if (colorMap.getDefaultBackground() != null || colorMap.getDefaultForeground() != null) {
+                    // The default color scheme will be opened automatically at the beginning of the stream on the next
+                    // line, so we don't want to duplicate it.
+                    nextOpenTags.remove(0);
```

Comment:
`.subList(1)` might be less scary to readers, though I see that the getter does make a fresh copy.

Reply:
Good point. I tried switching to `sublist`, but the implementation is not serializable so it would require another copy, which seems excessive, so I added a comment to help clarify for future readers but left the original code.